src/assets/js/customizer-controls/contextual-customizer-controls.js

```diff
@@ -466,6 +466,26 @@
                 });
             });
 
+            //add event handler to detect when opening sound is changed and play the sound to allow admin to preview it
+            $('select[data-customize-setting-link*=optin_sound]').on('change', function (event) {
```

Comment:
When a custom sound url is specified, try to trigger the sound too.

Reply:
@ShoboySnr you didnt fix this

Reply:
@w3guy i did fix it. When the custom audio url is entered, it get its value with the selector